lib/cn/cn_tree_compact.h

```diff
@@ -134,7 +133,7 @@ struct cn_compaction_work {
     struct kvs_rparams *     cw_rp;
     struct kvs_cparams *     cw_cp;
 
-    /* initialized in constructor (cn_tree_find_compaction_candidate) */
+    /* initialized in sp3_work() */
```

Comment:
I bet these comments about where things are initialized are not 100% accurate.

Reply:
I just hate these comments :)

Reply:
Nope, I would prefer to remove them...

Reply:
Do it

Reply:
Done.